src/components/ratings_reviews/ReviewModal.jsx

```diff
@@ -11,88 +12,126 @@ function ReviewModal() {
   // };
 
   return (
-    <section>
-      Rating
-      <ClickStars />
+    <div className="full-modal-container">
       <div
-        className="rating-definition"
+        className="modal-overlay"
+        onClick={toggleModal}
+        role="button"
+        tabIndex={0}
+        onKeyDown={toggleModal}
+        aria-label="close modal with overlay"
+      />
+      <div
+        className="outline-color"
         style={{
-          fontSize: '0.5em',
+          backgroundColor: 'cyan',
+          border: 'solid white 1px',
+          position: 'absolute',
+          padding: '1em',
+          alignSelf: 'center',
         }}
       >
-        <p>5 = Amazing</p>
-        <p>4 = Good</p>
-        <p>3 = Average</p>
-        <p>2 = Not terrible</p>
-        <p>1 = Bad</p>
-      </div>
-      <div className="do-you-recommend">
-        <p>Do you recommend this product?</p>
-        <input
-          type="radio"
-          value="Yes"
-          name="recommend-btn"
-          className="recommendation-radio-btns"
-        />
-        Yes
-        <input
-          type="radio"
-          value="No"
-          name="recommend-btn"
-          className="recommendation-radio-btns"
-        />
-        No
+        <section
+          className="reviewModal-container"
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            backgroundColor: 'white',
+            borderRadius: '5rem',
+            padding: '3em 1em 5em 1em',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+        >
+          Rating
+          <ClickStars />
```

Comment:
Not actually in this file, but in the ClickStars file can you please use nanoid to give each iterable a key?

`key=nanoid()`
`import { nanoid } from 'nanoid';`